google_compute_engine/network_setup/network_setup.py

```diff
@@ -130,7 +131,7 @@ def _EnableNetworkInterfaces(self, interfaces):
       except subprocess.CalledProcessError:
         self.logger.warning('Could not enable Ethernet interfaces.')
     else:
-      if os.path.exists('/etc/sysconfig/network-scripts'):
+      if os.path.exists(constants.LOCALBASE + '/etc/sysconfig/network-scripts'):
```

Comment:
If you rename the variable to `LOCAL_BASE`, this will be over 80 characters. I'd suggest creating a class variable (below network_interfaces) called `network_path` and then use `self.network_path` here and in `_DisableNetworkManager`. Given that both use these string constants, it's better to do that anyway.